lib/styleguide.js

```diff
@@ -28,12 +28,12 @@ function findVariationFiles(styleguide) {
 * Grabs variations of all components and returns an object organized by
 * components and their variations
 *
-* @param {String} styleguide - name of the styleguide that is set in the sites' config
+* @param {String} [styleguide] - name of the styleguide that is set in the sites' config
 * @returns {Object} componentVariations - an object that is organized by
 * component and its variations
 */
-function getVariations(styleguide) {
-  const foundVariations = styleguide ? findVariationFiles(styleguide) : [],
+function getVariations(styleguide = '_default') {
+  const foundVariations = styleguide !== '_default' ? findVariationFiles(styleguide).concat(findVariationFiles('_default')) : findVariationFiles('_default'),
```

Comment:
why are these being combined? In case there are default variations? If thats the case, can there be a code comment?

Reply:
good point, adding a comment